packages/dashboard/tests/frontend_selenium/tests/test_twin.py

```diff
@@ -1,102 +1,69 @@
 from utils.utils import generate_leters, generate_string, get_seed
 from pages.twin import TwinPage
-from pages.polka import PolkaPage
+from utils.grid_proxy import GridProxy
+from pages.dashboard import DashboardPage
 
 #  Time required for the run (6 cases) is approximately 3 minutes.
 
 
-def before_test_setup(browser, create_account):
+def before_test_setup(browser):
     twin_page = TwinPage(browser)
-    polka_page = PolkaPage(browser)
-    user = generate_string()
+    dashboard_page = DashboardPage(browser)
     password = generate_string()
-    polka_page.load_and_authenticate()
-    if (create_account):
-        polka_page.add_account(user, password)
-    else:
-        polka_page.import_account(get_seed(), user, password)
-    twin_page.navigate(user)
-    return twin_page, polka_page, password
+    dashboard_page.open_and_load()
+    dashboard_page.import_account(get_seed())
+    dashboard_page.connect_your_wallet(password).click()
+    twin_page.navigate()
+    return twin_page
 
 
-def test_accept_terms_conditions(browser):
+def test_twin_details(browser):
     """
-      Test Case: TC924 - Accept terms and conditions
-      Steps:
-          - Navigate to dashboard.
-          - Click on the desired account from the dashboard homepage.
-          - Click on Accept The Terms and Conditions.
-          - Use polka password authentication.
-      Result: Open same account on dashboard homepage and assert that no terms to accept when you come back to this account twin page.
-    """
-    twin_page, polka_page, password = before_test_setup(browser, True)
-    twin_page.accept_terms_conditions()
-    polka_page.authenticate_with_pass(password)
-    assert twin_page.wait_for('Accepted!')
-    assert twin_page.wait_for('Choose a Relay Address')
-
-
-def test_create_twin_relay(browser):
-    """
-      Test Case: TC932- check Balance
-      Test Case: TC1461 - Choose a Relay Address
```

Comment:
As mentioned in the `test_hompage` comment, twin creation should be covered.

Reply:
https://github.com/threefoldtech/tfgrid-sdk-ts/pull/903#discussion_r1288618109